src/snowflake/snowpark/mock/_snowflake_data_type.py

```diff
@@ -344,6 +349,7 @@ def __init__(
             {} if not sf_types_by_col_index else sf_types_by_col_index
         )
         self._null_rows_idxs_map = {}
+        self.sorted_by = []
```

Comment:
how is this information used?

Reply:
This metadata allows the `handle_order_by_clause` to annotate a TableEmulator with the columns that it hsa been sorted by. That is useful because some of the rank functions need that information to break ties if the data is sorted.

Reply:
oh yes, I saw that piece of information used by the _rank method